packages/outline-view/src/browser/outline-view-tree.ts

```diff
@@ -0,0 +1,40 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { Disposable } from '@theia/core/src/common';
+import { CompositeTreeNode, TreeModelImpl, TreeSelectionService, TreeExpansionService } from '@theia/core/lib/browser';
+
+@injectable()
+export class OutlineViewTreeModel extends TreeModelImpl {
+
+    @inject(TreeSelectionService) protected readonly selectionService: TreeSelectionService;
+    @inject(TreeExpansionService) protected readonly expansionService: TreeExpansionService;
+
+    handleExpansionChanged(): Disposable {
```

Comment:
it should be:
```ts
protected selectOnExpansion(node: ExpandableTreeNode): void {
   // don't select anything in order to preserve focus on current editor
}

Reply:
Thank you Anton! Just fixed it according to your advice.